ClockDetection.h

```diff
@@ -0,0 +1,19 @@
+#ifndef CLOCK_DETECTION_H
+#define CLOCK_DETECTION_H
+
+#include <iostream>
+#include <opencv2/opencv.hpp>
+
+using namespace std;
+using namespace cv;
+
+class ClockDetection {
+public:
+    /*
+    @breif convert colored image to greyscale
+    @param const cv::Mat& - color image 
+    */
+    cv::Mat convertToGray(const cv::Mat& inputImage);
+};
```

Comment:
What about he private variables? The image path could be the private variable, and the default constructor could read in the image using openCV.

Reply:
I added image path has a private variable and the default constructor reads in the input image.